src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -16,7 +19,9 @@ public class JwtTokenReader : IJwtTokenReader
         private const string MicrosoftDiscoveryEndpoint =
             "https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration";
 
-        private readonly string _applicationId;
+        private const string AudienceKeyName = "aud";
```

Comment:
What about using the `JwtClaimTypes.Audience` constant which exists in [here](https://github.com/IdentityModel/IdentityModel/blob/main/src/JwtClaimTypes.cs)

Reply:
done